Musicplayer.css

```diff
@@ -134,29 +165,28 @@ ul li ul li:hover{background: black}
 	width: 100%;
 	display: grid;
     grid-gap: 2%;
-	grid-template-columns: 2% 14% 14% 14% 14% 14% 14%;
+	grid-template-columns: 6% 14% 14% 14% 14% 12% 12%;
```

Comment:
21/9: I fix some of the position of the title so it match within the page so I don't need to high scroll bar(side way). Also setting fonts for it which the font that I decided to apply after text in the login page and because of I want my content to be consistence I apply the same font.